lib/index.js

```diff
@@ -3,11 +3,9 @@
 var pkg = require('../package');
 
 exports.versions = {
-  'copilot-util': pkg.version,
-  'immutable': pkg.dependencies.immutable
+  'copilot-util': pkg.version
```

Comment:
I'm not sure that this `versions` object has much value, tbh. maybe we can get rid of it in another PR?